coopr-docs/docs/source/installation/standalone.rst

```diff
@@ -32,7 +32,7 @@ for the UI, and Ruby 1.9.0p0 or higher for the provisioner. The server runs an e
 removing the need to setup a database or zookeeper instance. It is not recommended to run Coopr in this mode for anything other
 than experimentation.
 
-`Download <http://repository.cask.co/downloads/co/cask/coopr/coopr-standalone/0.9.8/coopr-standalone-0.9.8.zip>`_ the standalone zip.
+`Download <http://repository.cask.co/downloads/co/cask/coopr/coopr-standalone/0.9.9/coopr-standalone-0.9.9.zip>`_ the standalone zip.
```

Comment:
Looks like we need a Sphinx extension to handle this. I don't see a way to handle links that change with a release automatically.